components/ActivityCardPreview/index.js

```diff
@@ -40,3 +40,8 @@ export default function ActivityCardPreview({ activity, onToggleBookmark }) {
 const StyledImage = styled(Image)`
   border-radius: 10px;
 `;
+
+const StyledCardTitle = styled.h3`
```

Comment:
having this as an h2, as it was previously, is actually the better choice to keep your HTML semantic.

In your application you want to have only 1 h1, and the next most important headings should be h2. If it's too big and therefore you chose to go with an h3, you should adjust the styling.